python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -27,6 +33,12 @@ def _setupGangaPath():
 
 
 def start_ganga(gangadir_for_test, extra_opts=[]):
+    """
+    Startup Ganga by calling the same set of 'safe' functions each time
+    Args:
+        gangadir_for_test (str): This is the directory which the GangaUnitTest is to be run, creates a new gangadir per test to avoid collisions
```

Comment:
*This* function does not create a new gangadir per test. The calling function is responsible for that.